.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on: [push]
```

Comment:
On pull_request is better as the ci will then also work with forks.

Reply:
You're too fast xD
I only did that for testing

Reply:
Is it okay like this? Or do you want pull_request only?
`on: [push, pull_request]`